bitswap/src/behaviour.rs

```diff
@@ -224,9 +224,9 @@ impl NetworkBehaviour for Bitswap {
     fn inject_connected(&mut self, peer_id: &PeerId) {
```

Comment:
nit: since `PeerId` is now `Copy`, we might want to pass it by value later on

Reply:
I think this is the NetworkBehaviour trait impl, but PeerIds are larger than pointers (std::mem::size_of says 80 bytes), so not sure of the principle.